### DISCUSSION TITLE
how can i compile bcmwl-kernel-source with akms in alpine linux?

### DISCUSSION THREAD

**Discussion by creatxrgithub** [pgj/freebsd-wifibox#72] [Q&A]
i am tring alpine linux these days. 

in other distros, i can compile bcmwl-kernel-source with dkms to get my broadcom wifi card's driver.

how can i compile bcmwl-kernel-source with akms in alpine linux?

do you use bcmwl-kernel-source to let bcm43142 working with freebsd?

thanks.

[compile.log](https://github.com/pgj/freebsd-wifibox-alpine/files/13310804/compile.log)

**Comment by pgj** [comment]
As far as I am aware, Alpine Linux does not provide a package for Broadcom `wl` driver.  This means if you need that feature you will have to implement that yourself probably.  On the contrary, Wifibox/Alpine has the `broadcom-wl` package but it is geared towards the Wifibox environment and it does not do DKMS for example.

While there is no support provided for making `broadcom-wl` work on Alpine Linux here (it is out of scope), let me note that in the past few months there were some fixed made in that department and users could use BCM43xx cards with the recent versions (i.e. 1.3.0) of the `net/wifibox` port.
